GooglePlayPlugins/com.google.play.billing/Samples/TrivialKart/scripts/GasStorePageController.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class GasStorePageController : MonoBehaviour
```

Comment:
If yours class is not gonna to override Start() and Update(), do you have to use a MonoBehaviour?

Reply:
I am overriding Awake() and OnEnable() here, so I think MonoBehaviour is necessary.